sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/decimalExpressions.scala

```diff
@@ -128,7 +128,7 @@ case class PromotePrecision(child: Expression) extends UnaryExpression {
 case class CheckOverflow(
```

Comment:
Note: we need to change CheckOverflowInSum as well. However, the error context is actually empty even when WSCG is available. 
I need more time for that. I am making this one to catch up the Spark 3.3 RC2, which is happening soon.